docs/platforms/python/integrations/logging/index.mdx

```diff
@@ -72,8 +72,9 @@ sentry_sdk.init(
     # ...
     integrations=[
         LoggingIntegration(
-            level=logging.INFO,        # Capture info and above as breadcrumbs
-            event_level=logging.INFO   # Send records as events
+            level=logging.INFO,                 # Capture INFO and above as breadcrumbs
+            event_level=logging.INFO,           # Send INFO records as events
+            sentry_logs_level=logging.WARNING,  # Capture WARNING and above as logs
```

Comment:
I'd suggest setting these to more realistic levels; specifically, it seems unrealistic to me that someone would want their `event_level` below the `sentry_logs_level`

Maybe set `level` to debug, `event_level` to warning, and `sentry_logs_level` also to debug?

Reply:
Or, alternatively, it might be helpful to just demonstrate setting each of the levels to their respective defaults; that way, folks are not confused into thinking that these values are the default values.